packages/user/src/views/EmailVerificationReminder.tsx

```diff
@@ -24,6 +29,12 @@ const EmailVerificationReminder = () => {
       });
   };
 
+  useEffect(() => {
+    if (user?.isEmailVerified) {
+      navigate("/");
```

Comment:
`/` may not always be the home route. We have `getHomeRoute` helper function to determine the correct home route. lets use that one.